cssom-view-1/Overview.bs

```diff
@@ -1556,7 +1556,7 @@ The <dfn attribute for=HTMLElement>offsetParent</dfn> attribute must return the
     1. If <var>ancestor</var> is <a>closed-shadow-hidden</a> from the element and its computed value of the 'position' property is ''position/fixed'', terminate this algorithm and return null.
     1. If <var>ancestor</var> is not <a>closed-shadow-hidden</a> from the element and satisfies at least one of the following, terminate this algorithm and return <var>ancestor</var>.
         * The element is a containing block of absolutely-positioned descendants (regardless of whether there are any absolutely-positioned descendants).
-        * The element has a non-default used value of 'zoom'.
+        * The element has a different [=effective zoom=] than its parent.
```

Comment:
This is not right, is it? You want something like: ` * <var>ancestor</var> has a different [=effective zoom=] than the element.` Or ` * The element has a different [=effective zoom=] than <var>ancestor</var>`